azure/src/azure-account.api.d.ts

```diff
@@ -41,7 +41,7 @@ export interface AzureSession {
 
 export interface AzureSubscription {
 	readonly session: AzureSession;
-	readonly subscription: SubscriptionModels.Subscription;
+	readonly subscription: Subscription;
```

Comment:
I'm less sure about this change. We're going from the type in `@azure/arm-resources-subscriptions@1.0.0` (which I think is track 1) to `@azure/arm-resources-subscriptions@2.0.0` (which I think is track 2). But neither of those match what's actually exported by the Azure Account extension, which is from `@azure/arm-subscriptions@3.1.2` (which I think is track 1). In any case, the objects _look_ identical but...I dunno.

Reply:
Interesting... I need to look into this more before stamping a "should work just fine" on it 😄 

> But neither of those match what's actually exported

What do you mean by "match"? The types are incompatible? 

I wonder why haven't we moved to the arm-subscriptions T2 SDK yet? 🤔

Reply:
By match, I just mean, is not from the same package. Azure Account is using `@azure/arm-subscriptions@3.1.2`.

Reply:
Seems ok since that's how it was before. That said, I would like to know why we're using `@azure/arm-subscriptions@3.1.2` in Azure Account, but `@azure/arm-resources-subscriptions` here.

Reply:
🤷